test/SymbolGraph/Symbols/Mixins/DeclarationFragments/Full/NominalTypes.swift

```diff
@@ -6,11 +6,19 @@
 // RUN: %FileCheck %s --input-file %t/NominalTypes.symbols.json --check-prefix=ENUM
 // RUN: %FileCheck %s --input-file %t/NominalTypes.symbols.json --check-prefix=TYPEALIAS
 
-public struct S<T> where T: Sequence {}
+@frozen public struct S<T> where T: Sequence {}
```

Comment:
Should we also test when a declaration has multiple attributes?

Reply:
It wouldn't hurt.